docs/02_cleos/02_how-to-guides/how-to-get-transaction-information.md

```diff
@@ -1,20 +1,192 @@
 ## Goal
 
-Query infomation of a transaction
+Query infomation of an EOSIO transaction
 
 ## Before you begin
 
-* Install the currently supported version of `cleos`
 
-* Understand the following:
 
-  * What is a transaction
 
-## Steps
+* Install the currently supported version of `cleos`.
+
+[[info | Note]]
+| `cleos` is bundled with the EOSIO software. [Installing EOSIO](../../00_install/index.md) will also install `cleos`.
+
+
+
+* Understand how transactions work in an EOSIO blockchain. For more information on transactions, see the [Transactions Protocol](https://developers.eos.io/welcome/latest/protocol-guides/transactions_protocol) section.  
 
+## Steps
+Run the following command to query information of an executed transaction:
 ```sh
-cleos get transaction id
+cleos get transaction <id>
+```
+Where `id` is the transaction ID of the executed transaction. 
```

Comment:
if user takes it literally he will run the command like this
`cleos get transaction <870a6b6e3882061ff0f64016e1eedfdd9439e2499bf978c3fb29fcedadada9b1>`

I would re-phrase it
"Assuming you want to get transaction with id 870a6b6e3882061ff0f64016e1eedfdd9439e2499bf978c3fb29fcedadada9b1 run the following command:
cleos get transaction 870a6b6e3882061ff0f64016e1eedfdd9439e2499bf978c3fb29fcedadada9b1
"